disnake/components.py

```diff
@@ -279,8 +276,47 @@ def to_dict(self) -> SelectMenuPayload:
         return payload
 
 
+class SelectMenu(BaseSelectMenu):
+    """Represents a string select menu from the Discord Bot UI Kit.
+
+    .. versionadded:: 2.0
```

Comment:
this is missing the user-constructable line that previously existed.

Reply:
https://github.com/DisnakeDev/disnake/pull/800/commits/46c30453d0d22f6c4c4a2ce5eb481dcea9ba9941